src/vulkan/pipeline.cc

```diff
@@ -409,6 +423,20 @@ Result Pipeline::SendDescriptorDataToDeviceIfNeeded() {
       }
     }
 
+    // Initialize transfer buffers. Transfer images are already initialized in
```

Comment:
Why do we need to initialize them in different places?

Reply:
Transfer buffers needed to be initialized in later phase, because the buffer usage flags could change while looping through the descriptors in the loop above. Both transfer buffers and images are now initialized here to make the code more readable.